src/scheduler/scheduler.ts

```diff
@@ -15,7 +15,7 @@ export enum SchedulerMode {
 
 export class SchedulerConfig {
     crontab: string = '* * * * *'; // Every minute
-    mode ?: SchedulerMode = SchedulerMode.ExactlyOncePerInterval;
+    mode ?: SchedulerMode = SchedulerMode.ExactlyOncePerIntervalWhenActive;
```

Comment:
To completely eliminate the breaking change, I'm going to revert the communicator retries change and instead log exceptions retried by communicators so they aren't swallowed.

Reply:
For the scheduled workflows, it never seems to have worked right anyways, so I'm fine changing the default.

Reply:
I don't want to have no default because that's an even bigger breaking change, and now this PR has no breaking changes at all (except to what's already broken).